en_US/performance/benchmark.md

```diff
@@ -2,9 +2,46 @@
 
 [emqtt_bench](https://github.com/emqx/emqtt_bench) is a concise and powerful MQTT protocol performance testing tool written with Erlang. If you need testing services with large-scale scenarios and in-depth customization , the test service [XMeter](https://www.xmeter.net/) is recommended.
 
-## Compile and install
+## Docker image
```

Comment:
```suggestion
## Install Test Tool
### Run Docker Image
```

Reply:
I unserstand there are thress options to install emqtt-bench. My previous suggestion to make the structure looks like the following. What do you think?

- Install Test Tool
   - Run Docker Image
   - Download Binary Package
   - Build from Source
- Use Test Tool
   - Publish
   - Subscribe
   - Connect
   - ...
- Typical Stress Test Scenario
   - ...